<launch>
    <include file="$(find object_detection)/launch/object_recognition.launch"/>
    <node name="main_state" pkg="main_state" type="main.py" output="screen" args="TestObjectRecogAndManip" />
</launch>